apps/cowswap-frontend/src/modules/tradeFormValidation/services/validateTradeForm.ts

```diff
@@ -88,5 +91,14 @@ export function validateTradeForm(context: TradeFormValidationContext): TradeFor
     return TradeFormValidation.ApproveRequired
   }
 
+  if (
+    !isWrapUnwrap &&
+    derivedTradeState.tradeType !== TradeType.LIMIT_ORDER &&
+    !tradeQuote.isLoading &&
+    isQuoteExpired(tradeQuote.response?.expiration) === true
+  ) {
+    return TradeFormValidation.QuoteExpired
+  }
```

Comment:
Don't show this:
1. For wraps
2. When quote is loading
3. When trade type is LIMIT